R/network_analysis.R

```diff
@@ -0,0 +1,96 @@
+#' Analyse protein interaction network for significant hits
+#'
+#' The STRING database provides a resource for known and predicted protein-protein interactions. The type of
+#' interactions include direct (physical) and indirect (functional) interactions. Through the R package
+#' \code{STRINGdb} this resource if provided to R users. This function provides a convenient wrapper for
+#' \code{STRINGdb} functions that allow an easy use within the protti pipeline.
+#'
+#' @param data A data frame that contains significantly changing proteins (STRINGdb is only able to plot
+#' 400 proteins at a time so do not provide more for network plots). Information about treatment binding can be provided and will be displayed as colorful halos around
+#' the proteins in the network.
+#' @param protein_id The name of the column containing the protein accession numbers.
+#' @param string_id The name of the column containing STRING database identifiers. These can be obtained from UniProt.
+#' @param organism_id Numeric organism ID (NCBI taxon-ID). This can be obtained from \href{https://string-db.org/cgi/input?sessionId=bpvps5GS2As6&input_page_show_search=on}{here}.
+#' H. sapiens: 9606, S. cerevisiae: 4932, E. coli: 511145.
+#' @param score_threshold The interaction score based on \href{https://string-db.org/cgi/info?sessionId=bBP5N4cIf0PA&footer_active_subpage=scores}{STRING}
+#' has to be between 0 and 1000. A score closer to 1000 is related to a higher confidence for the interaction. The default value is 900.
+#' @param binds_treatment The name of the column containing a logical indicating if the corresponding
```

Comment:
is the logical "is_known" as in the example? Maybe you should specify this

Reply:
is_known is a column that contains logicals.